### PR TITLE
modify the error version of k8s

### PR DESCRIPTION
**Pull request by timyinshi**
Signed-off-by: timyinshi <shiguangyin@inspur.com>

/kind bug
/sig docs

modify the error version 1.7 to 1.17

### CODE REVIEW

File: content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md [hunk 1]
```diff
@@ -28,23 +28,23 @@ We’re pleased to announce the delivery of Kubernetes 1.17, our fourth and fina
 <!--
 ### Cloud Provider Labels reach General Availability
 -->
-作为公开测试版特性添加到v1.2，v1.7中可以看到云提供商标签达到基本可用。
+作为公开测试版特性添加到v1.2，v1.17中可以看到云提供商标签达到基本可用。
```
Comment (tengqm) [inline content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md:31]: is it 1.2 or 1.12?

File: content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md [hunk 2]
```diff
@@ -28,23 +28,23 @@ We’re pleased to announce the delivery of Kubernetes 1.17, our fourth and fina
...
 <!--
 Added as a beta feature way back in v1.2, v1.17 sees the general availability of cloud provider labels.
 -->
 ### 卷快照进入公开测试版
 <!--
 ### Volume Snapshot Moves to Beta
 -->
-在v1.7中，Kubernetes卷快照特性是公开测试版。这个特性是在v1.12中以内部测试版引入的，第二个有重大变化的内部测试版是v1.13。
+在v1.17中，Kubernetes卷快照特性是公开测试版。这个特性是在v1.12中以内部测试版引入的，第二个有重大变化的内部测试版是v1.13。
```
Comment (tanjunchen) [inline content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md:39]: 中英文与数字之间加个空格
```suggestion
在 v1.17 中，Kubernetes 卷快照特性是公开测试版。这个特性是在 v1.12 中以内部测试版引入的，第二个有重大变化的内部测试版是 v1.13。
```

File: content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md (hunk 1 above)
Comment (timyinshi) [reply content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md]: it is 1.2

File: content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md (hunk 2 above)
Comment (timyinshi) [reply content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md]: 已经增加

**Comment by tengqm** [comment]
the '1.2' looks a typo to me. however, this is sync from English content.
/lgtm
/approve

**Pull request merged**